sqlglot/dialects/clickhouse.py

```diff
@@ -332,6 +332,8 @@ class Parser(parser.Parser):
             TokenType.SET,
         }
 
+        RESERVED_TOKENS = {*parser.Parser.RESERVED_TOKENS} - {TokenType.SELECT}
```

Comment:
you don't need to make a copy just to RESERVED_TOKENS - ...

Reply:
Good catch, thanks